receiver/httpcheckreceiver/testdata/expected_metrics/endpoint_404.yaml

```diff
@@ -19,82 +19,82 @@ resourceMetrics:
               dataPoints:
                 - asInt: "0"
                   attributes:
-                    - key: http.url
-                      value:
-                        stringValue: http://invalid-endpoint
-                    - key: http.status_code
-                      value:
-                        intValue: "404"
                     - key: http.method
                       value:
                         stringValue: GET
                     - key: http.status_class
                       value:
                         stringValue: 1xx
+                    - key: http.status_code
```

Comment:
I'd like to see what it looks like to rewrite all golden metrics files with this sortation applied. I think the easiest way to do it would be to temporarily hack the `ReadMetrics` function, so that it will read -> sort -> rewrite. Then you can just run tests and the files should be updated.